source/shared/cpp/ObjectModel/Column.h

```diff
@@ -45,6 +45,7 @@ class Column : public BaseCardElement
 
     std::string m_width;
     std::vector<std::shared_ptr<AdaptiveSharedNamespace::BaseCardElement>> m_items;
+    static const std::unordered_map<CardElementType, std::function<std::shared_ptr<BaseCardElement>(const Json::Value&)>, EnumHash> CardElementParsers; // I dont know what this is used for
```

Comment:
Did this come from an earlier version of the change? Shouldn't be needed, please remove.